### PR TITLE
add write_registers to climate target temperature

### PR DESCRIPTION
**Pull request by brunoenten**
## Proposed change

In the climate integration, add the possibility to use write_registers not only for hvac_mode like before, but also for target temperature, using separate config attribute.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/97475
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Review by janiversen — DISMISSED** [review]
LGTM.

**Review by janiversen — DISMISSED** [review]

**Review by frenck — CHANGES_REQUESTED** [review]
👋  Hi @brunoenten,

There is a merge conflict with this PR. Could you please take a look?

Thanks! 👍 

../Frenck

**Comment by janiversen** [comment]
@brunoenten be aware the documentation have been rewritten, so your changes now goes into different lines.

**Review by janiversen — DISMISSED** [review]
LGTM, Thanks

File: source/_integrations/modbus.markdown [hunk 1]
```diff
@@ -712,6 +712,11 @@ climates:
       description: "Register address for target temperature (Setpoint)."
       required: true
       type: integer
+    target_temp_write_registers:
+      description: if true use write_registers for target temperature
```
Comment (frenck) [inline source/_integrations/modbus.markdown]:
```suggestion
      description: "If `true` use `write_registers` for target temperature."
```

**Review by frenck — APPROVED** [review]
Thanks, @brunoenten 👍 

../Frenck

**Pull request merged**